sup3r/cli.py

```diff
@@ -289,6 +290,58 @@ def valid_pipeline_keys(ctx):
     ctx.invoke(pipeline_keys)
 
 
+@main.group(invoke_without_command=True)
+@click.option('--dry-run', is_flag=True,
+              help='Flag to do a dry run (make batch dirs without running).')
+@click.option('--cancel', is_flag=True,
+              help='Flag to cancel all jobs associated with a given batch.')
+@click.option('--delete', is_flag=True,
+              help='Flag to delete all batch job sub directories associated '
+              'with the batch_jobs.csv in the current batch config directory.')
+@click.option('--monitor-background', is_flag=True,
+              help='Flag to monitor all batch pipelines continuously '
+              'in the background using the nohup command. Note that the '
+              'stdout/stderr will not be captured, but you can set a '
+              'pipeline "log_file" to capture logs.')
+@click.option('-v', '--verbose', is_flag=True,
+              help='Flag to turn on debug logging.')
+@click.pass_context
+def batch(ctx, dry_run, cancel, delete, monitor_background, verbose):
+    """Create and run multiple sup3r project directories based on batch
+    permutation logic.
+
+    The sup3r batch module (built on the reV batch functionality) is a way to
```

Comment:
Cool idea. What are you using this for?

Reply:
all sorts of stuff, you can use it to test different models or fwp chunk sizes+overlaps, or to set up jobs for multiple years.

Reply:
Yeah multiple years is what I though of first. Looks good!